api/models/Membership.js

```diff
@@ -58,6 +58,9 @@ module.exports = bookshelf.Model.extend(merge({
     .save({}, {transacting: opts.transacting})
     .tap(() => User.followDefaultTags(userId, communityId, opts.transacting))
     .tap(() => Community.find(communityId, {transacting: opts.transacting})
+      .tap(community => community.save({
+        num_members: community.get('num_members') + 1
```

Comment:
i guess not using [increment](http://knexjs.org/#Builder-increment) here is ok because you already have the community instance... but one thing to be aware of is that by default, the save operation will create a query that updates all the fields in the model, not just the one you changed. if you just want that, you have to use `patch`:

```js
community.save({num_members: 7}, {patch: true, transacting}) // note it's in a separate object
```